app/views/candidate_mailer/application_on_pause.erb

```diff
@@ -1,21 +1,15 @@
 Dear <%= @application_form.first_name %>,
```

Comment:
`def referees_did_not_respond` for this email?

Reply:
`referees_did_not_respond_before_end_of_cycle` following discussion with @EmmaFrith